src/js/models/search/queryBuilders/ContractFilterQuery.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
What is this file being used for/by?

Reply:
@kevinli-work It isn't.  Missed deleting when I moved these query builders to the `OtherFiltersQuery` file.  Removing.